<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QnRecordingBusinessActionWidget</class>
 <widget class="QWidget" name="QnRecordingBusinessActionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Quality:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="qualityComboBox"/>
   </item>
   <item>
    <widget class="QSpinBox" name="fpsSpinBox">
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="value">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="fpsLabel">
     <property name="text">
      <string>fps</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="beforeLabel">
     <property name="text">
      <string>before</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="beforeSpinBox">
     <property name="suffix">
      <string>s</string>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="afterLabel">
     <property name="text">
      <string>Post-recording:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="afterSpinBox">
     <property name="suffix">
      <string>s</string>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="durationSpinBox">
     <property name="suffix">
      <string>s</string>
     </property>
     <property name="maximum">
      <number>86400</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="durationLabel">
     <property name="text">
      <string>max</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
